api/server/server.go

```diff
@@ -77,9 +77,8 @@ func (s *Server) Close() {
 	}
 }
 
-// serveAPI loops through all initialized servers and spawns goroutine
-// with Serve method for each. It sets createMux() as Handler also.
-func (s *Server) serveAPI() error {
+// ServeAPI starts listening for inbound requests.
+func (s *Server) ServeAPI() error {
```

Comment:
🚲 🏠 as this is exporting a method that wasn't exported before;  would naming this just `Serve()` work? (called as `api.Serve()` or `server.Serve()`

(not a blocker, but "api server serve API" felt a bit awkward)